src/sources/redis/list.rs

```diff
@@ -46,6 +61,11 @@ impl InputHandler {
     }
 }
 
+async fn backoff_exponential(exp: &u32) {
+    let ms = if *exp <= 9 { 2_u64.pow(*exp) } else { 1000 };
```

Comment:
Can we start at 32 and move up from there?

```suggestion
    let ms = if exp <= 4 { 2_u64.pow(exp + 5) } else { 1000 };
```